data/assets/examples/renderable/renderablecartesianaxes/cartesianaxes.asset

```diff
@@ -0,0 +1,29 @@
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The 
+-- parent is set to Earth which means the center of the axes will be at the center of Earth.
+
+local Node = {
+  Identifier = "Renderable_Cartesian_Axes_Example",
+  Transform = {
```

Comment:
If this is a basic example do we want a `StaticScale` here? Since we want these to be as minimum as possible, or did we say that `StaticScale` is an exception?